spec/controllers/comments_controller_spec.rb

```diff
@@ -15,6 +15,13 @@
         xhr :post, :create, { question_id: question.id, comment: comment.attributes }
       }.to change(Comment, :count).by(1)
     end
+
+    it "回答に投稿すると、コメントが1つ追加されること" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.